### PR TITLE
Add view and serializer for PostalCodeArea

### PR DESCRIPTION
**Pull request by juuso-j**
# View and Serializer for PostalCodeArea

### Breakdown

1. munigeo/api.py
    * Added serializer and view for PostalCodeAreas and postalcodearea field for Addresses.

### CODE REVIEW

File: munigeo/api.py [hunk 1]
```diff
@@ -360,6 +360,49 @@ def get_queryset(self):
 register_view(AdministrativeDivisionViewSet, 'administrative_division')
 
 
+class PostlCodeSerializer(TranslatedModelSerializer):
```
Comment (mhieta) [inline munigeo/api.py:363]: Fix typo here

File: munigeo/api.py [hunk 2]
```diff
@@ -421,7 +464,11 @@ def to_representation(self, obj):
         div_qs = AdministrativeDivisionGeometry.objects.select_related(
             "division", "division__type"
         ).filter(boundary__contains=obj.location, division__type__type="postcode_area")
-        ret['postal_code'] = div_qs.first().division.name if div_qs else None
+        if obj.postal_code_area:
+            ret['postal_code'] = obj.postal_code_area.postal_code
+        else:
+            ret['postal_code'] = div_qs.first().division.name if div_qs else None
+        ret['postal_code_area'] = PostlCodeSerializer(obj.postal_code_area).data
```
Comment (mhieta) [inline munigeo/api.py:471]: Typo here also

**Review by mhieta — CHANGES_REQUESTED** [review]
Fix typos. Otherwise looks good.

File: munigeo/api.py (hunk 1 above)
Comment (juuso-j) [reply munigeo/api.py]: Fixed.

File: munigeo/api.py (hunk 2 above)
Comment (juuso-j) [reply munigeo/api.py]: Fixed.

**Review by mhieta — APPROVED** [review]
LGTM 👍

**Pull request merged**